ui/scripts/vpc.js

```diff
@@ -745,12 +745,19 @@
                         }
                     },
                     dataProvider: function(args) {
+                        var data = {
+                            page: args.page,
+                            pageSize: pageSize,
+                            networkid: args.context.networks[0].id,
+                            listAll: true
+                        };
+                        var keyword = (((args || {}).filterBy || {}).search || {}).value;
+                        if (keyword) {
+                            data.keyword = keyword;
+                        }
                         $.ajax({
                             url: createURL('listLoadBalancers'),
-                            data: {
-                                networkid: args.context.networks[0].id,
-                                listAll: true
-                            },
+                            data: data,
                             success: function(json) {
```

Comment:
@rhtyd Changed the code. Eliminated the array1.join()

Reply:
Fantastic, good work @vedulasantosh I'll wait for another LGTM/testing/screenshot and we can merge this then.